delivery_schenker/models/delivery_carrier.py

```diff
@@ -419,6 +420,10 @@ def _schenker_shipping_information_without_packages_volume(self, picking):
         )
 
     def _schenker_shipping_information_without_packages(self, picking):
+        if not any(
+            not move_line.result_package_id for move_line in picking.move_line_ids
```

Comment:
not any not = all

to make it easier to read :)

Reply:
oh yes :-)